src/controllers/IndexController.php

```diff
@@ -131,7 +131,7 @@ class="fb-cta cta--yellow">
       $next_game_text = tr('Soon');
       $countdown = array('--', '--', '--', '--');
     } else {
-      $next_game_text = date(tr('date and time format'), $next_game);
+      $next_game_text = date('H:i:s T D m/d/y', $next_game);
```

Comment:
This makes fbctf less customizable.

if we are not using a translation here, a constant string containing the format should be added to the controller ( e.g `IndexController::NEXT_GAME_DATE_FORMAT` )